.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": false
```

Comment:
You should set this to `true` ;)
Can we remove this file from version control?

Reply:
Oops, didn't mean to include that file. 
I didn't want to setup linting, since I don't normally program in Python. It was giving import errors.
I've removed the file.